devicon.json

```
@@ -5956,6 +5956,26 @@
                 "alias": "plain-wordmark"
             }
         ]
+    },
+        {
+        "name": "ngrx",
```

The altnames attribute is missing
```suggestion
        "name": "ngrx",
        "altnames": [],
```